feature-detects/css/hyphens.js

```diff
@@ -52,19 +52,22 @@ define(['Modernizr', 'prefixes', 'createElement', 'testAllProps', 'addTest'], fu
           var result = false;
           var firstChild = document.body.firstElementChild || document.body.firstChild;
 
+          /* Hyphenation is only applied when language is explicitly set and when respective dictionary
+           * is available. See https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens for details. */
+          div.lang = 'en';
```

Comment:
good find. just for documentation purposes: does it fix any special issue (in this github repo or in your work)?

Reply:
I think I was puzzled why the test page reports unsupported hyphenation on Safari and FF while they both actually support it.